examples/example_wallet_electrum/src/main.rs

```diff
@@ -92,5 +97,29 @@ fn main() -> Result<(), anyhow::Error> {
     client.transaction_broadcast(&tx)?;
     println!("Tx broadcasted! Txid: {}", tx.compute_txid());
 
+    println!("=== Performing Partial Sync ===");
+    std::io::stdout().flush().expect("must flush");
+    let sync_request =
+        wallet
+            .start_sync_with_revealed_spks()
+            .inspect(|sync_item, sync_progress| {
+                let progress_percent =
+                    (100 * sync_progress.consumed()) as f32 / sync_progress.total() as f32;
+                eprintln!("[ SCANNING {:03.0}%] {}", progress_percent, sync_item)
```

Comment:
Good! I personally think this approach is good, but I'm wondering if it's too verbose.

Reply:
Okay